client/python/tests/test_facet_v2.py

```diff
@@ -202,3 +202,39 @@ def set_test_schemaURL(self):  # noqa: N802
             expected_event = json.load(f)
 
         assert expected_event == event_sent
+
```

Comment:
Might want to check what happens if you `del` additional property

Reply:
I changed the behaviour now. I don't think there is a need to test that when creating new class with attr's public methods.